### PR TITLE
Show organized campaigns on My Dashboard

### PR DESCRIPTION
**Pull request by MusikAnimal**
Bug: https://phabricator.wikimedia.org/T152742

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -110,6 +111,17 @@ def student?(course)
     course.users.role('student').include? self
   end
 
+  def campaign_organizer?
+    organized_campaigns.any?
+  end
+
+  def organized_campaigns
```
Comment (ragesoss) [inline app/models/user.rb]: Why not just use `.campaigns` in place of this? We don't have any other roles at this point, and if/when we do, we may still want to show all a user's campaigns.

**Comment by ragesoss** [comment]
@sejalkhatri would you like to review this?

File: app/models/user.rb (hunk 1 above)
Comment (MusikAnimal) [inline app/models/user.rb]: So should this be moved to DashboardPresenter? There we want to show only campaigns the current user organizes. I just thought it was a nifty method that could exist on the model

Comment (ragesoss) [inline app/models/user.rb]: My point is that, until/if we have another role, user.organized_campaigns will be the same as user.campaigns. So why not just use user.campaigns in the view or presenter or wherever, and dispense with this method entirely?

**Comment by sejalkhatri** [comment]
Ya sure, m on it!

File: app/views/campaigns/_campaigns.html.haml [hunk 2]
```diff
@@ -47,34 +47,7 @@
...
-              = campaign_presenter.word_count
-          %td
-            %span.views-human
-              = number_to_human(campaign_presenter.courses.sum(:view_sum))
-            %span.views.hidden
-              = campaign_presenter.courses.sum(:view_sum)
-          %td
-            %span.students
-              = campaign_presenter.campaign.students_without_nonstudents.count
+        = render 'campaigns/row', presenter: campaign_presenter, campaign: campaign
```
Comment (sejalkhatri) [inline app/views/campaigns/_campaigns.html.haml]: If this is the only place we are using this view then shouldn't we put the entire loop(lines 48/49) in that?

Comment (MusikAnimal) [inline app/views/campaigns/_campaigns.html.haml]: It is also used in [dashboard/index.html.haml](https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/1110/files#diff-6ba95d8ef933598e22ac06d1e009b57eR135). The convention of having `_row.html.haml` just be the `<tr>` markup itself is modeled after the [courses/row](https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/app/views/courses/_row.html.haml) partial

Comment (sejalkhatri) [inline app/views/campaigns/_campaigns.html.haml]: ohk! that makes sense.

File: spec/features/dashboard_spec.rb [hunk 3]
```diff
@@ -146,4 +146,27 @@
       expect(page).to have_content 'Recent Title'
     end
   end
+
+  context 'campaigns', focus: true do
```
Comment (sejalkhatri) [inline spec/features/dashboard_spec.rb]: focus: true -- should be removed I guess

Comment (MusikAnimal) [inline spec/features/dashboard_spec.rb]: whoops! Thank you :)

**Comment by sejalkhatri** [comment]
everything else looks good to me!

**Comment by MusikAnimal** [comment]
Thanks!

**Pull request merged**